### PR TITLE
The Moneytizer Bid Adapter: initial release

### PR DESCRIPTION
**Pull request by themoneytizer**
## Type of change

- [x] New bidder adapter   

## Description of change

Initial PR for The Moneytizer bid adapter

- contact email of the adapter’s maintainer : tech@themoneytizer.com
- test parameters for validating bids:
```
{
  bidder: 'themoneytizer',
  params: {
    pid: -1,
    test: 1
  }
}
```

## Other information

- Documentation PR: https://github.com/prebid/prebid.github.io/pull/5129

### CODE REVIEW

File: modules/themoneytizerBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+```
+Module Name: The Moneytizer Bid Adapter
+Module Type: Bidder Adapter
+Maintainer: tech@themoneytizer.com
+```
+
+## Description
+
+Module that connects to The Moneytizer demand sources
```
Comment (ncolletti) [inline modules/themoneytizerBidAdapter.md:11]: could we include a table explaining all possible params that your adapter accepts: pid, test, placement, formats, isInternal, baseUrl

File: modules/themoneytizerBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    const response = serverResponse.body;
+
+    if (response && response.bid && !response.timeout && !!response.bid.ad) {
+      bidResponses.push(response.bid);
+    }
+
+    return bidResponses;
+  },
+  getUserSyncs: function (syncOptions, serverResponses) {
+    if (!syncOptions.iframeEnabled) {
```
Comment (ncolletti) [inline modules/themoneytizerBidAdapter.js:90]: Add check for `syncOptions.pixelEnabled`

File: modules/themoneytizerBidAdapter.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        sizes: [[300, 250]],
+      },
+    },
+    bids: [
+      {
+        bidder: "themoneytizer",
+        params: {
+          pid: -1,
+          test: 1
+        },
```
Comment (ncolletti) [inline modules/themoneytizerBidAdapter.md:31]: update with params that return valid bid responses from your endpoint, including user sync data in `body.c_sync`

File: modules/themoneytizerBidAdapter.md (hunk 1 above)
Comment (themoneytizer) [reply modules/themoneytizerBidAdapter.md:11]: - The 'placement, formats, and isInternal' were for development only and have been removed

- Table added with pid, test and baseUrl

File: modules/themoneytizerBidAdapter.js (hunk 2 above)
Comment (themoneytizer) [reply modules/themoneytizerBidAdapter.js]: check added https://github.com/prebid/Prebid.js/pull/11047/commits/8fd19c45d17d9363a0c6da04e113e0b56363faf8

File: modules/themoneytizerBidAdapter.md (hunk 3 above)
Comment (themoneytizer) [reply modules/themoneytizerBidAdapter.md:39]: Only `pid` and `test` params return valid bid responses. `baseUrl` is unused currently and will be custom for specific publishers. Should I remove it for now?

user sync data added in `body.c_sync` ✅

Comment (ncolletti) [reply modules/themoneytizerBidAdapter.md:39]: No problem to keep baseUrl.

For the test params, I am seeing an error returned:
```
Warning: Undefined property: stdClass::$schain in /var/www/html/m/class/client/parserClient.class.php on line 61
{"ads":"no-ads","err-msg":"E9"}
```

Comment (themoneytizer) [reply modules/themoneytizerBidAdapter.md:39]: should be fixed now

**Review by ncolletti — APPROVED** [review]
looks good!

**Pull request merged**